postgres/README.md

```diff
@@ -103,9 +103,12 @@ Follow the instructions below to configure this check for an Agent running on a
 
 **Available for Agent >6.0**
 
-PostgreSQL default logging is to `stderr` and logs do not include detailed information. It is recommended to log into a file with additional details specified in the log line prefix.
+PostgreSQL default logging is to `stderr` and logs do not include detailed information. It is recommended to log into 
+a file with additional details specified in the log line prefix.
```

Comment:
There is no limitation on number of char per line, we should avoid having break lines in middle of sentences.